dags/openshift_nightlies/config/install/rogcp/ovn-411.json

```diff
@@ -0,0 +1,12 @@
+{
+    "rosa_environment": "staging",
+    "ocm_environment": "stage",
+    "openshift_worker_count": 27,
+    "ocm_cli_version": "container",
+    "managed_channel_group": "candidate",
+    "managed_ocp_version": "4.11.0-fc.3-candidate",
+    "openshift_network_type": "OVNKubernetes",
+    "openshift_worker_instance_type": "custom-8-32768",
+    "machineset_metadata_label_prefix": "machine.openshift.io",
+    "openshift_workload_node_instance_type": "custom-16-65536"
+}
```

Comment:
Just curious, I would usually do such version specific json in my playground and keep upstream only pointed to latest to avoid unnecessary DAGs in production(that takes longer to load the UI and most of them are paused), is there any particular reason to pin this release in upstream ?

Reply:
Im not a fan of having airflow playgrounds, I only use it to develop new features and then, I remove the branch and the playground. but its ok, I have removed the fixed version dag